src/main/java/com/elixr/poc/service/PurchaseService.java

```diff
@@ -0,0 +1,60 @@
+package com.elixr.poc.service;
+import com.elixr.poc.data.Purchase;
+import com.elixr.poc.rest.request.PurchaseRequest;
+import com.elixr.poc.rest.response.PurchaseErrorResponse;
+import com.elixr.poc.rest.response.PurchaseResponse;
+import com.elixr.poc.rest.validator.PurchaseValidator;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import java.util.List;
+import java.util.UUID;
+
+/* service class for editing response and save or update new purchase to database  */
+@Service
+public class PurchaseService {
+    public ResponseEntity<?> editResponse(PurchaseRequest newPurchase) {
+        PurchaseResponse newPurchaseResponse;
+        Purchase newPurchaseFormat = createUserObjectFromRequest(newPurchase);
```

Comment:
I would suggest renaming this method to relate to purchase.

Reply:
updated